### PR TITLE
Added operator contract balance checks to the pricing test script

### PR DESCRIPTION
**Pull request by pdyraga**
We can use operator contract balance check to make sure the value of
rewards stored in the contract matches the total ETH balance stored on
the contract.

### CODE REVIEW

**Comment by ngrinkevich** [comment]
That looks much cleaner than mine 😄

What do you think of adding an error message just in case there's a mismatch? i.e.

```
if (!operatorContractBalance.eq(rewardsSum.add(dkgSubmitterReimbursementFee))) {
    console.log("ERROR: Operator Contract balance mismatch");
}
```

File: solidity/scripts/pricing-test.js [hunk 1]
```diff
@@ -44,6 +44,42 @@ module.exports = async function() {
...
+
+            let rewardsSum = web3.utils.toBN(0);
+            for (let i = 0; i < accounts.length; i++) {
+                rewardsSum = rewardsSum.add(web3.utils.toBN(prevRewards[i]));
+            }
+
+            const serviceContractBalance = await web3.eth.getBalance(contractService.address);
+            const dkgFeePool = await contractService.dkgFeePool();
+            const requestSubsidyFeePool = await contractService.requestSubsidyFeePool();
+            const dkgContributionMargin = await contractService.dkgContributionMargin();
```
Comment (pdyraga) [inline solidity/scripts/pricing-test.js:56]: Those functions are not currently exposed on the service contract. I'll add them in a separate PR. Let's hold on with merging this one.

Comment (pdyraga) [reply solidity/scripts/pricing-test.js:56]: Done in https://github.com/keep-network/keep-core/pull/1738

**Comment by pdyraga** [comment]
Working on this one. Should have it ready pretty soon.

**Comment by pdyraga** [comment]
We now have both service and operator contract checks so I put them in the table:

image

**Comment by pdyraga** [comment]
One sec, something is wrong here...

**Pull request merged**